perfkitbenchmarker/vm_util.py

```diff
@@ -525,25 +525,30 @@ def IssueCommand(cmd, force_info_log=False, suppress_warning=False,
   logging.info('Running: %s', full_cmd)
 
   shell_value = RunningOnWindows()
-  process = subprocess.Popen(cmd, env=env, shell=shell_value,
-                             stdin=subprocess.PIPE, stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE)
+  with tempfile.TemporaryFile() as tf_stderr:
+    with tempfile.TemporaryFile() as tf_stdout:
```

Comment:
LGTM. Minor style nitpick.

Python 2.7 supports multiple context manager in the 'with' statement.
https://docs.python.org/dev/whatsnew/2.7.html#other-language-changes

So, you could simplify it something like:

```python
with tempfile.TemporaryFile() as tf_stderr, tempfile.TemporaryFile() as tf_stdout:
  ...
```